templates/incl-editor-and-output.html

```diff
@@ -106,3 +106,16 @@
       </div>
     </div>
   </div>
+<!-- developer mode toggle -->
+<div class="mx-auto w-64 border-gray-400 border-2 rounded-full py-2 px-4">
+  <label for="developers_toggle" class="flex flex-row items-center justify-center cursor-pointer">
+    <div class="relative">
+      <input id="developers_toggle" type="checkbox" onclick="hedyApp.toggle_developers_mode()" class="sr-only" />
+      <div class="w-10 h-4 bg-gray-400 rounded-full shadow-inner"></div>
+      <div class="dot absolute w-6 h-6 bg-white rounded-full shadow -left-1 -top-1 transition" style="top: -5px;"></div>
+    </div>
+    <div class="ml-3">
+      Developers mode
```

Comment:
Shall we add this to texts and localize it? I guess for Dutch kids this is ok but for Greek or Chinese kids we want the option to show it in their own lang.

Reply:
Will add a localization based on the *.yaml files and will let you know when it's finished.

Reply:
Fixed!